### ISSUE TITLE
How to enter a valid token?

### ISSUE THREAD

**Issue by Captain-Rage** [joaopsys/AutoChronoGG#2]
When following the instructions

```
[autochronogg@Bumpty ~]$ ./chronogg.py 
<<<AutoChronoGG>>>
Usage: ./chronogg.py <Cookie>
Please insert your cookie. Press CTRL+SHIFT+J on the website (CTRL+SHIFT+K on Firefox) and type document.cookie. Then paste the whole string as an argument.
You only need need to do this once because AutoChronoGG will remember your cookie (if valid).
```

this message is thrown:
 
```
Fetching https://api.chrono.gg/quest/spin
Error processing webpage: HTTP Error 401: Unauthorized
An error occurred while fetching results (probably expired/invalid cookie). Terminating...

```

I tried following the instructions about the XHR string also but that didn't work either. Do I have to browse the page once in a webbrowser on the same machine as I execute the script for this to work?

**Comment by joaopsys** [comment]
Hi,

No, you shouldn't need to do it on the same machine. I've just followed my README to check if it was still up to date, and it worked for me.

Make sure you're viewing the request headers. I've attached a screenshot of this. The field you will need to copy and feed to AutoChronoGG is the 'Authorization' that starts with JWT (that included). I didn't include all of it for obvious reasons.

![image](https://user-images.githubusercontent.com/1836514/35249255-987b6a12-ffd1-11e7-9b79-840c9fbe33fc.png)
